packages/okta-react/test/jest/auth.test.js

```diff
@@ -122,4 +135,27 @@ describe('Auth component', () => {
       foo: 'bar'
     });
   });
+  test('returns isAuthenticated true when TokenManager can get the accessToken', async () => {
```

Comment:
Suggested wording, `'isAuthenticated() returns true when the TokenManager returns an access token'`